package.json

```diff
@@ -21,8 +21,13 @@
   },
   "dependencies": {
     "@nestjs/common": "^10.0.0",
+    "@nestjs/config": "^3.2.3",
     "@nestjs/core": "^10.0.0",
+    "@nestjs/mongoose": "^10.0.10",
     "@nestjs/platform-express": "^10.0.0",
+    "class-validator": "^0.14.1",
+    "dotenv": "^16.4.5",
```

Comment:
why using dotenv library? nest use that package in their own packages for the config service